### PR TITLE
Add `$landscape` and `$portrait` stores to `trackScreenOrientation` integration

### PR DESCRIPTION
**Pull request by efremenkovan**

### CODE REVIEW

File: .changeset/clean-geese-cheat.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@withease/web-api': patch
+---
+
+Add `$isLandscapeOrientation` and `$isPortraitOrientation` stores to `trackScreenOrientation` integration
```
Comment (igorkamyshev) [inline .changeset/clean-geese-cheat.md:5]: 🤓 for me, the next spell `trackScreenOrientation(...).$isLandscapeOrientation` is a little bit excessive. Could we name it like `$landscape`?

File: .changeset/clean-geese-cheat.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@withease/web-api': patch
```
Comment (igorkamyshev) [inline .changeset/clean-geese-cheat.md:2]: 🚨 why it is considered as a patch? As I understand, it is a new feature, and it has to be released as `minor`.

File: .changeset/clean-geese-cheat.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+---
+
```
Comment (igorkamyshev) [inline .changeset/clean-geese-cheat.md:4]: 🚨  I do not see any updates in `website/docs`. In my opinion, it's required to update docs before release. So, it's a blocker.

**Review by igorkamyshev — APPROVED** [review]
Thanks for the contribution 😇 I've left a couple messages, can you fix it please before merging?

File: .changeset/clean-geese-cheat.md (hunk 3 above)
Comment (efremenkovan) [reply .changeset/clean-geese-cheat.md:4]: [Updated](https://github.com/igorkamyshev/withease/pull/70/commits/f3f7563189d677fa9e36aba75091416780c49254) the documentation and demo to contain information and samples of new stores

File: .changeset/clean-geese-cheat.md (hunk 1 above)
Comment (efremenkovan) [reply .changeset/clean-geese-cheat.md]: [Renamed](https://github.com/igorkamyshev/withease/pull/70/commits/02c71287e8e965944e64a68e1ec75295c44116e6) new stores to suggested format

**Pull request merged**